ext/qiita_marker/qfm_custom_block.c

```diff
@@ -92,10 +92,11 @@ static cmark_node *try_opening_qfm_custom_block_block(
         1, sizeof(node_qfm_custom_block));
 
     cmark_strbuf *info = parser->mem->calloc(1, sizeof(cmark_strbuf));
+    bufsize_t info_startpos = cmark_parser_get_first_nonspace(parser) + matched;
```

Comment:
[memo]
The number of characters before the opening custom block fence should also be considered.

Example,

```
> :::foo bar
  |  |
  └--|-- first_nonspace
     └-- info_startpos
  |-|
  matched
```